plugins/src/webrtcsink/imp.rs

```diff
@@ -187,19 +191,19 @@ enum ControllerType {
     // Running the "loss based controller"
     Loss,
 }
-
 struct Consumer {
     pipeline: gst::Pipeline,
     webrtcbin: gst::Element,
+    rtprtxsend: Option<gst::Element>,
     webrtc_pads: HashMap<u32, WebRTCPad>,
     peer_id: String,
     encoders: Vec<VideoEncoder>,
-    /// None if congestion control was disabled
+    // Our Homegrown controller
```

Comment:
This is all pretty awkward, I think we should move the implementation of the homegrown CC to a separate file, and name things more explicitly in here, the current state is confusing wrt the gcc controller that exists as an element.

Reply:
Well, moving it away is a bit of work which is not really needed or can be done later fmpov

Reply:
Well I disagree :)

Reply:
Moved it into its own file (in a dedicated commit)